packages/protocol/contracts/governance/LockedGold.sol

```diff
@@ -178,12 +178,18 @@ contract LockedGold is
     Balances storage account = balances[msg.sender];
     // Prevent unlocking gold when voting on governance proposals so that the gold cannot be
     // used to vote more than once.
-    require(!getGovernance().isVoting(msg.sender), "Account locked");
+    uint256 remainingLockedGold = getAccountTotalLockedGold(msg.sender).sub(value);
+
+    uint256 totalReferendumVotes = getGovernance().getAmountOfGoldUsedForVoting(msg.sender);
```

Comment:
Minor naming nit: maybe `totalGovernanceVotes`? Since it's both referendum and upvotes.